modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -148,16 +149,18 @@ public void tearDown() throws Exception {
 	public void testGetObjectDefinition() throws Exception {
 		super.testGetObjectDefinition();
 
-		ObjectDefinition objectDefinition =
-			testGetObjectDefinitionsPage_addObjectDefinition(
+		ObjectDefinition objectDefinitionA =
+			objectDefinitionResource.postObjectDefinition(
 				randomObjectDefinition());
 
 		String objectDefinitionPluralName = StringUtil.lowerCaseFirstLetter(
-			TextFormatter.formatPlural(objectDefinition.getName()));
+			TextFormatter.formatPlural(objectDefinitionA.getName()));
 
 		Assert.assertEquals(
 			"/o/c/" + objectDefinitionPluralName,
-			objectDefinition.getRestContextPath());
+			objectDefinitionA.getRestContextPath());
```

Comment:
@victorKammerer let's keep this part of the test as it was before, no need to change.